src/main/kotlin/app/revanced/patches/googlephotos/preferences/ShowBackUpWhileChargingPatch.kt

```diff
@@ -0,0 +1,33 @@
+package app.revanced.patches.googlephotos.preferences
+
+import app.revanced.util.exception
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
+import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.googlephotos.preferences.fingerprints.BackupPreferencesFingerprint
+import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
+
+@Patch(
+    name = "Show hidden 'Back up while charging' toggle",
+    description = "This toggle is hidden by default, when enabled, backups run only when the device is charging",
+    compatiblePackages = [CompatiblePackage("com.google.android.apps.photos")]
+)
+@Suppress("unused")
+object BackupPreferencesFingerprint : BytecodePatch(
+    setOf(BackupPreferencesFingerprint)
+) {
+    override fun execute(context: BytecodeContext) {
+        // patches 'backup_prefs_had_backup_only_when_charging_enabled' to always be true
```

Comment:
Follow this comment style:

```suggestion
        // Patches 'backup_prefs_had_backup_only_when_charging_enabled' to always be true.
```